sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1353,6 +1354,8 @@ def deinit(self):
         if self.y_cable_presence[0] is True:
             y_cable_helper.delete_ports_status_for_y_cable()
 
+        del platform_chassis
```

Comment:
Please delete the platform object rather than platform_chassis. In the sequence we create platform API which internally creates platform_chassis. So during cleanup we should delete platform object. If the object is not cached please cache it and use it during deletion.